osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -66,7 +66,10 @@ protected APIScoresCollection Scores
                 if (value?.Scores.Any() != true)
                     return;
 
-                scoreManager.OrderByTotalScoreAsync(value.Scores.Select(s => s.CreateScoreInfo(rulesets, Beatmap.Value.ToBeatmapInfo(rulesets))).ToArray(), loadCancellationSource.Token)
+                // TODO: temporary. should be removed once `OrderByTotalScore` can accept `IScoreInfo`.
+                var beatmapInfo = new BeatmapInfo { MaxCombo = Beatmap.Value.MaxCombo };
```

Comment:
Navigated deep at the components that receive scores associated with this temporary `BeatmapInfo`, and it appears one of them actually use `Score.BeatmapInfo.Status` to determine whether to show the PP column (see [usage](https://github.com/ppy/osu/blob/853cf6feaa165e833ecb7ca18c6cdffe8ca6e005/osu.Game/Overlays/BeatmapSet/Scores/TopScoreStatisticsSection.cs#L118)).
```suggestion
                var beatmapInfo = new BeatmapInfo 
                {
                    Status = Beatmap.Value.Status,
                    MaxCombo = Beatmap.Value.MaxCombo,
                };
```

Reply:
Nice catch.